### DISCUSSION TITLE
[Projects Beta] Showing Labels in Board Views

### DISCUSSION THREAD

**Discussion by bpmooch** [community/community#14918] [Projects and Issues] [answered]
Is it possible to show labels in board views? I created a roadmap project and would like to show what labels are applied to each issue on the board

**Comment by schalkneethling** [comment]
@bpmooch, you can add this as another column inside a view by using the `+`(plus - Add field) button and selecting `labels` from the drop-down.

![Screenshot showing a project board with the add field drop-down expanded. The drop-down shows various options such as title, assignee, etc.](https://user-images.githubusercontent.com/10350960/163548901-37909c51-4b84-4a10-87c7-53311be866b8.png)

**Reply by bpmooch** [reply]
This makes sense for table views, but how can I show labels on tickets in a board view? Specifically, I want the same view as what is available on the [github roadmap](https://github.com/orgs/github/projects/4247)

**Comment by 2percentsilk** [comment, marked as answer]
Hi @bpmooch!

You'll need to set up edit your configuration (from your response above, I'm assuming you already have a table view set up bte). Try these steps and let me know if you have any questions!
1. Click the drop down menu next to the name of your view.
2. Under `Configuration` select the first option. This should be `Title, Assignees, and Status` - this item indicates what is currently part of your view.
    ![image](https://user-images.githubusercontent.com/3174849/163851441-32aa098e-2b34-4951-8bfd-fe3ba9101cf3.png)
3. Check the box by `Labels` to add labels to your view
  ![image](https://user-images.githubusercontent.com/3174849/163851574-1df0b382-2da6-47e2-90ea-c608716205bc.png)
4. You should see the labels appear automatically!
5. Be sure to save your changes :) 

Hope that helps!

**Reply by bpmooch** [reply]
thanks!

**Reply by roenc576** [reply]
This helped me too, thank you so much!!!!
